inreach-parser/inreach-parser.bicep

```diff
@@ -0,0 +1,177 @@
+}
+
+resource bikejamboree_ftp 'Microsoft.Web/sites/basicPublishingCredentialsPolicies@2022-09-01' = {
+  parent: bikejamboree_site
+  name: 'ftp'
+  location: location
+  tags: {
+    'hidden-link: /app-insights-resource-id': insights_id
+  }
+  properties: {
+    allow: true
+  }
+}
+
+resource bikejamboree_scm 'Microsoft.Web/sites/basicPublishingCredentialsPolicies@2022-09-01' = {
+  parent: bikejamboree_site
+  name: 'scm'
+  location: location
+  tags: {
+    'hidden-link: /app-insights-resource-id': insights_id
+  }
+  properties: {
+    allow: true
+  }
+}
+
+resource bikejamboree_web 'Microsoft.Web/sites/config@2022-09-01' = {
+  parent: bikejamboree_site
+  name: 'web'
+  location: location
+  tags: {
+    'hidden-link: /app-insights-resource-id': insights_id
+  }
+  properties: {
+    numberOfWorkers: 1
+    defaultDocuments: [
+      'Default.htm'
+      'Default.html'
+      'Default.asp'
+      'index.htm'
+      'index.html'
+      'iisstart.htm'
+      'default.aspx'
+      'index.php'
+    ]
+    netFrameworkVersion: 'v4.0'
+    linuxFxVersion: 'Node|16'
+    requestTracingEnabled: false
+    remoteDebuggingEnabled: false
+    httpLoggingEnabled: false
+    acrUseManagedIdentityCreds: false
+    logsDirectorySizeLimit: 35
+    detailedErrorLoggingEnabled: false
+    publishingUsername: '$bikejamboree-inreach-parser'
+    scmType: 'None'
+    use32BitWorkerProcess: false
+    webSocketsEnabled: false
+    alwaysOn: false
+    managedPipelineMode: 'Integrated'
+    virtualApplications: [
+      {
+        virtualPath: '/'
+        physicalPath: 'site\\wwwroot'
+        preloadEnabled: false
+      }
+    ]
+    loadBalancing: 'LeastRequests'
+    experiments: {
+      rampUpRules: []
+    }
+    autoHealEnabled: false
+    vnetRouteAllEnabled: false
+    vnetPrivatePortsCount: 0
+    cors: {
+      allowedOrigins: [
+        'https://portal.azure.com'
+      ]
+      supportCredentials: false
+    }
+    localMySqlEnabled: false
+    ipSecurityRestrictions: [
+      {
+        ipAddress: 'Any'
+        action: 'Allow'
+        priority: 2147483647
+        name: 'Allow all'
+        description: 'Allow all access'
+      }
+    ]
+    scmIpSecurityRestrictions: [
+      {
+        ipAddress: 'Any'
+        action: 'Allow'
+        priority: 2147483647
+        name: 'Allow all'
+        description: 'Allow all access'
+      }
+    ]
+    scmIpSecurityRestrictionsUseMain: false
+    http20Enabled: false
+    minTlsVersion: '1.2'
+    scmMinTlsVersion: '1.2'
+    ftpsState: 'FtpsOnly'
+    preWarmedInstanceCount: 0
+    functionAppScaleLimit: 200
+    functionsRuntimeScaleMonitoringEnabled: false
+    minimumElasticInstanceCount: 0
+    azureStorageAccounts: {}
+  }
+}
+
+resource bikejamboree_domain 'Microsoft.Web/sites/hostNameBindings@2022-09-01' = {
+  parent: bikejamboree_site
+  name: '${function_name}.azurewebsites.net'
+  location: location
+  properties: {
+    siteName: function_name
+    hostNameType: 'Verified'
+  }
+}
```

Comment:
Z tego, co rozumiem to chyba jest wygenerowane automatycznie z apki? Tak 3/4 tego można wyrzucić, ale z drugiej strony, pewnie nie ma co się męczyć

BTW, Node.JS w wersji 16 będzie mieć EOL niedługo, o tym ostrzega panel Azure

Reply:
Tak, generowane.
Podmieniłem na `Node|18`